<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="109.641" tests="9" failures="3">
  <testsuite name="Root Suite" timestamp="2023-08-17T10:44:56" tests="0" file="cypress/e2e/Ventes/Clients.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Clients" timestamp="2023-08-17T10:44:56" tests="9" time="109.589" failures="3">
    <testcase name="Clients Get_ExportClients_Excel Separtor COMMA 200" time="0.000" classname="Get_ExportClients_Excel Separtor COMMA 200">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: GET
URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&amp;exportFile%5Btype%5D=CSV&amp;exportFile%5Bseparator%5D=COMMA

No response was received within the timeout.

https://on.cypress.io/request" type="CypressError"><![CDATA[CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: GET
URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&exportFile%5Btype%5D=CSV&exportFile%5Bseparator%5D=COMMA

No response was received within the timeout.

https://on.cypress.io/request
    at <unknown> (http://localhost:42645/__cypress/runner/cypress_runner.js:132377:70)
    at tryCatcher (http://localhost:42645/__cypress/runner/cypress_runner.js:1807:23)
    at <unknown> (http://localhost:42645/__cypress/runner/cypress_runner.js:4186:41)
    at tryCatcher (http://localhost:42645/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:42645/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:42645/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:42645/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:42645/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:42645/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:42645/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:42645/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:42645/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.ee (webpack://api/./node_modules/cypress-plugin-api/dist/support.js:14:2002)]]></failure>
    </testcase>
    <testcase name="Clients Get_ExportClients_Excel 200" time="0.000" classname="Get_ExportClients_Excel 200">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: GET
URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&amp;exportFile%5Btype%5D=EXCEL

No response was received within the timeout.

https://on.cypress.io/request" type="CypressError"><![CDATA[CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: GET
URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&exportFile%5Btype%5D=EXCEL

No response was received within the timeout.

https://on.cypress.io/request
    at <unknown> (http://localhost:42645/__cypress/runner/cypress_runner.js:132377:70)
    at tryCatcher (http://localhost:42645/__cypress/runner/cypress_runner.js:1807:23)
    at <unknown> (http://localhost:42645/__cypress/runner/cypress_runner.js:4186:41)
    at tryCatcher (http://localhost:42645/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:42645/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:42645/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:42645/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:42645/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:42645/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:42645/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:42645/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:42645/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.ee (webpack://api/./node_modules/cypress-plugin-api/dist/support.js:14:2002)]]></failure>
    </testcase>
    <testcase name="Clients Get_ExportClients_Excel exportFile 403" time="0.000" classname="Get_ExportClients_Excel exportFile 403">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: GET
URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&amp;exportFile%5Btype%5D=CSV&amp;exportFile%5Bseparator%5D=

No response was received within the timeout.

https://on.cypress.io/request" type="CypressError"><![CDATA[CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: GET
URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&exportFile%5Btype%5D=CSV&exportFile%5Bseparator%5D=

No response was received within the timeout.

https://on.cypress.io/request
    at <unknown> (http://localhost:42645/__cypress/runner/cypress_runner.js:132377:70)
    at tryCatcher (http://localhost:42645/__cypress/runner/cypress_runner.js:1807:23)
    at <unknown> (http://localhost:42645/__cypress/runner/cypress_runner.js:4186:41)
    at tryCatcher (http://localhost:42645/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:42645/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:42645/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:42645/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:42645/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:42645/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:42645/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:42645/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:42645/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.ee (webpack://api/./node_modules/cypress-plugin-api/dist/support.js:14:2002)]]></failure>
    </testcase>
    <testcase name="Clients Client registration France 201" time="4.810" classname="Client registration France 201">
    </testcase>
    <testcase name="Clients Client registration without_contact_email" time="1.343" classname="Client registration without_contact_email">
    </testcase>
    <testcase name="Clients Client registration without_contact_city" time="1.367" classname="Client registration without_contact_city">
    </testcase>
    <testcase name="Clients Client registration Clients_TOKEN_NOK" time="1.436" classname="Client registration Clients_TOKEN_NOK">
    </testcase>
    <testcase name="Clients Get_Client " time="3.648" classname="Get_Client ">
    </testcase>
    <testcase name="Clients Get_Client List " time="1.610" classname="Get_Client List ">
    </testcase>
  </testsuite>
</testsuites>